homeassistant/components/history_stats/helpers.py

```diff
@@ -72,39 +72,28 @@ def async_calculate_period(
     return start, end
 
 
-class HistoryStatsHelper:
-    """Static methods to make the HistoryStatsSensor code lighter."""
-
-    @staticmethod
-    def pretty_duration(hours):
-        """Format a duration in days, hours, minutes, seconds."""
-        seconds = int(3600 * hours)
-        days, seconds = divmod(seconds, 86400)
-        hours, seconds = divmod(seconds, 3600)
-        minutes, seconds = divmod(seconds, 60)
-        if days > 0:
-            return "%dd %dh %dm" % (days, hours, minutes)
-        if hours > 0:
-            return "%dh %dm" % (hours, minutes)
-        return "%dm" % minutes
-
-    @staticmethod
-    def pretty_ratio(value, period):
-        """Format the ratio of value / period duration."""
-        if len(period) != 2 or period[0] == period[1]:
-            return 0.0
-
-        ratio = 100 * 3600 * value / (period[1] - period[0]).total_seconds()
-        return round(ratio, 1)
-
-    @staticmethod
-    def handle_template_exception(ex, field):
-        """Log an error nicely if the template cannot be interpreted."""
-        if ex.args and ex.args[0].startswith("UndefinedError: 'None' has no attribute"):
-            # Common during HA startup - so just a warning
-            _LOGGER.warning(ex)
-            return
-        _LOGGER.error("Error parsing template for field %s", field, exc_info=ex)
+def pretty_duration(hours: float) -> str:
+    """Format a duration in days, hours, minutes, seconds."""
+    seconds = int(3600 * hours)
+    days, seconds = divmod(seconds, 86400)
+    hours, seconds = divmod(seconds, 3600)
+    minutes, seconds = divmod(seconds, 60)
+    if days > 0:
+        return "%dd %dh %dm" % (days, hours, minutes)
+    if hours > 0:
+        return "%dh %dm" % (hours, minutes)
```

Comment:
We normally use f-strings.

Reply:
This is relocated code which was later removed in a future cleanup iteration via #70720